src/Umbraco.Web.UI.Client/src/views/common/overlays/user/user.html

```diff
@@ -49,12 +49,11 @@ <h5>
 
         <div ng-repeat="login in externalLoginProviders">
 
-			<form ng-if="login.linkedProviderKey == undefined" method="POST" action="{{externalLinkLoginFormAction}}" id="oauthloginform" name="oauthloginform">
+			<form ng-submit="linkProvider($event)" ng-if="login.linkedProviderKey == undefined" method="POST" action="{{externalLinkLoginFormAction}}">
 				<input type="hidden" name="provider" value="{{login.authType}}" />
                 <button class="btn btn-block btn-social"
                         ng-class="login.properties.SocialStyle"
-                        id="{{login.authType}}"
-                        onclick="document.forms.oauthloginform.submit();">
+                        id="{{login.authType}}">
```

Comment:
Can we at least keep `name="oauthloginform"` here? And maybe use `id="oauthloginform-{login.linkedProviderKey}"` to make it unique for each form?

Reply:
I agree, I'll add this.